.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  # Maintain dependencies for python
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "monthly"
```

Comment:
I think the interval could be increased to weekly

Reply:
Okay, I can change and try it weekly to see how it works. 

But if we aren't doing very frequent releases and automated testing I don't think it will matter and will just be a lot of updates that we aren't deploying. We can see how it goes.